release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -990,6 +1028,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Prior to this relase, if the `LVMCluster` CR was created with an invalid device path in the `deviceSelector.optionalPaths` field , the `LVMCluster` CR was in `Progressing` state. With this release, If the `deviceSelector.optionalPaths` field contains an invalid device path, {lvms} updates the `LVMCluster` CR state to `Failed`. (link:https://issues.redhat.com/browse/OCPBUGS-23995[OCPBUGS-23995])
+
+* Prior to this release, upon updating {product-title}, the old LVM resource pods were also scheduled on the cluster along with the latest LVM resource pods. With this release, upon updating {product-title}, {lvms} configures the `priorityClassName` parameter to ensure that only the latest LVM resource pods are scheduled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23375[OCPBUGS-23375])
+
+* Prior to this release, upon creating the `LVMCluster` CR, {lvms} skipped the counting of the volume groups. This resulted in the `LVMCluster` CR moving to `Progressing` state even when the volume groups were valid. With this release, upon creating the `LVMCluster` CR, {lvms} counts all the volume groups, and updates the `LVMCluster` CR state to `Ready` if the volume groups are valid. (link:https://issues.redhat.com/browse/OCPBUGS-23191[OCPBUGS-23191])
+
+* Prior to this release, if the default device class was not present on all nodes, {lvms} failed to detect all the default device classes. With this release, {lvms} detects all the default device classes even if the default device class is present only on the selected nodes. With this update, you can define the default device class only on the selected nodes. (link:https://issues.redhat.com/browse/OCPBUGS-23181[OCPBUGS-23181])
+
+* Prior to this release, upon deleting the worker node in the single-node OpenShift (SNO) and worker node topology, the `LVMCluster` CR still included the configuration of the deleted worker node. This resulted in the `LVMCluster` CR remaining in `Progressing` state. With this release, upon deleting the worker node in the SNO and worker node topology, {lvms} deletes the worker node configuration in the `LVMCluster` CR, and updates the `LVMCluster` CR state to `Active`. (link:https://issues.redhat.com/browse/OCPBUGS-13558[OCPBUGS-13558])
+
+* Prior to this release, the Assisted Installer and the Assisted Service did not support installing {lvms} on multi-node clusters. With this release, the Assisted Installer and the Assisted Service support installing {lvms} on multi-node cluters. With this update, you can install {lvms} on the {product-title} cluster with multiple master nodes. (link:https://issues.redhat.com/browse/OCPBUGS-24319[OCPBUGS-24319])
```

Comment:
This needs to be removed as the errata had this removed due to the Bug not being validated in time

Reply:
Just to confirm:

Release note about "Assisted Installer" must be removed?

Reply:
Correct, respective bug is no longer part of errata due to timeline conflict.

Reply:
Updated as per the suggestion.